packages/go_router/lib/src/pages/custom_transition_page.dart

```diff
@@ -176,6 +176,7 @@ class NoTransitionPage<T> extends CustomTransitionPage<T> {
   }) : super(
           transitionsBuilder: _transitionsBuilder,
           transitionDuration: const Duration(microseconds: 1),
+          reverseTransitionDuration: const Duration(microseconds: 1),
```

Comment:
can you update both to duration.zero This seems like a oversight

Reply:
Sure, done.